hy/_compat.py

```diff
@@ -14,3 +15,33 @@ def reraise(exc_type, value, traceback=None):
         raise value.with_traceback(traceback)
     finally:
         traceback = None
+
+
+if PY3_9:
+
+    import ast, math, cmath
+    class Unparser(ast._Unparser):
+      # Work around some limitations of Python's `ast.unparse`.
```

Comment:
Add a reference to https://github.com/python/cpython/pull/24897. Since it's going to be backported to 3.9, we can eventually remove this workaround once it lands.

Reply:
Done.